### PR TITLE
Criteo:

### PR DESCRIPTION
**Pull request by numismatika**
- more subdomains

### CODE REVIEW

File: src/chrome/content/rules/Criteo.xml [hunk 1]
```diff
@@ -19,15 +19,1472 @@ Fetch error: http://pixel.criteo.net/ => https://pixel.criteo.net/: (60, 'SSL ce
 
 <ruleset name="Criteo (partial)" default_off='failed ruleset test'>
```
Comment (J0WI) [inline src/chrome/content/rules/Criteo.xml]: Please note that this rule is still `default_off`.

Please also add `^criteo.com`.

**Comment by J0WI** [comment]
This has been re-enabled in #12184. Would be great if you could rebase your brnach.

**Comment by numismatika** [comment]
@J0WI checks are green now

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**